src/ui/gridview/gridview.ios.ts

```diff
@@ -265,6 +265,7 @@ export default class GridViewIOS<TEvent extends string = GridViewEvents> extends
     this._refreshEnabled = value;
     if (value) {
       this.nativeObject.addSubview(this.refreshControl);
+      this.nativeObject.alwaysBounceVertical = true;
     } else {
       this.refreshControl.removeFromSuperview();
```

Comment:
Let's add this.nativeObject.alwaysBounceVertical = false; to reset the bounce feature @boztopuz